tests/factory_fixtures/lease.py

```diff
@@ -1,16 +1,18 @@
 import pytest
+from faker import Faker
 from models.lease import LeaseModel
 from datetime import datetime
 
 @pytest.fixture
 def lease_attributes():
     def _lease_attributes(unitNum, tenant, property):
+        fake = Faker()
         return {
             "unitNum": unitNum,
             "tenantID": tenant.id,
             "propertyID": property.id,
-            "dateTimeStart": datetime.utcnow(),
-            "dateTimeEnd": datetime.utcnow(),
+            "dateTimeStart": fake.date_time_this_decade(),
+            "dateTimeEnd": fake.date_time_this_decade(False, True), #the future
             "occupants": 3
```

Comment:
Very nice I like this. The comment is great, however passing false and true arguments without knowing what they're for is confusing. Perhaps it could be changed to do this? `fake.date_time_this_decade(before_now=False, after_now=True)` - Then the comment is not needed.

Will also want to use faker for the occupants field, and the unitNum field on line 22.